akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -8,11 +8,13 @@ import java.util.function.{ BiFunction, Function, Supplier }
 
 import akka.annotation.ApiMayChange
 import akka.annotation.InternalApi
+import akka.http.impl.util.JavaMapping
+import akka.http.javadsl.server.directives.{ AnyParamDirectives, FramedEntityStreamingDirectives, TimeoutDirectives }
 import akka.http.javadsl.server.directives.FramedEntityStreamingDirectives
 
 import scala.annotation.varargs
 
-abstract class AllDirectives extends FramedEntityStreamingDirectives
+abstract class AllDirectives extends AnyParamDirectives
```

Comment:
I think that one is wrong. You need to add the extension to BasicDirectives, instead and keep, WebSocketDirectives here (though, currently it is weird because someone broke up the alphabetic order here).